### PR TITLE
Feat/pm 704 GitHub config

### PR DESCRIPTION
**Pull request by skrimusss**
[PM-704](https://playmakerpro.atlassian.net/browse/PM-704)

added changes

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -42,7 +41,13 @@
...
     "react/jsx-no-bind": "off",
     "@typescript-eslint/no-use-before-define": "off",
-    "radix": "off"
+    "radix": "off",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ]
```
Comment (dejwi) [inline .eslintrc.json:50]: Why ? it doesn't change anything for me

File: tailwind.config.js [hunk 2]
```diff
@@ -1,93 +1,138 @@
...
+      'body-s': '0.75rem',
+      'body-xs': '0.625rem',
+      'label-2xl': '1.125rem',
+      'label-xl': '1rem',
+      'label-md': '0.875rem',
+      'label-s': '0.75rem',
+      'caption-xl': '0.875rem',
+      'caption-md': '0.75rem',
+      'caption-s': '0.625rem',
+    },
```
Comment (dejwi) [inline tailwind.config.js:122]: just click commit suggestion
```suggestion
    fontSize: {
      'heading-2xl': '2.5rem',
      'heading-xl': '2rem',
      'heading-md': '1.5rem',
      'heading-sm': '1.25rem',
      'heading-smaller': '1.125rem',
      'heading-xs': '1rem',
      'heading-micro': '0.875rem',
      'subHeading-2xl': '1.125rem',
      'subHeading-xl': '1rem',
      'subHeading-md': '0.875rem',
      'subHeading-sm': '0.75rem',
      'body-xl': '1rem',
      'body-md': '0.875rem',
      'body-sm': '0.75rem',
      'body-xs': '0.625rem',
      'label-2xl': '1.125rem',
      'label-xl': '1rem',
      'label-md': '0.875rem',
      'label-sm': '0.75rem',
      'caption-xl': '0.875rem',
      'caption-md': '0.75rem',
      'caption-sm': '0.625rem',
    },
```

**Review by dejwi — CHANGES_REQUESTED** [review]
why THIRD pr 💀

**Comment by skrimusss** [comment]
> why THIRD pr 💀

training bro 💀

File: .eslintrc.json (hunk 1 above)
Comment (skrimusss) [reply .eslintrc.json:50]: I got an error and I couldnt do anything about it only it helped I dont know what it depends on but without it jsx doesnt work as how it should work

Comment (dejwi) [reply .eslintrc.json:50]: oh I think it's because we're on diffrent operating systems and end line characters are diffrent, so okay

**Review by dejwi — APPROVED** [review]
Good job💯

and use only one pr for one task and remember rest of our gitworkflow stuff:)

**Pull request merged**